fem/tmop_tools.cpp

```diff
@@ -437,6 +437,7 @@ double TMOPNewtonSolver::ComputeScalingFactor(const Vector &x,
    const double norm_in = Norm(r);
 
    const double detJ_factor = (solver_type == 1) ? 0.25 : 0.5;
+   compute_metric_quantile_flag = false;
```

Comment:
Shouldn't these be updated during the line search? They affect the energy and the residual.